sonarqube/resource_sonarqube_github_binding.go

```diff
@@ -17,7 +17,7 @@ type GetBinding struct {
 	Alm                   string `json:"alm"`
 	Repository            string `json:"repository"`
 	URL                   string `json:"url"`
-	SummaryCommentEnabled bool   `json:"summaryCommentEnabled"`
+	SummaryCommentEnabled bool   `json:"summaryCommentEnabled,omitempty"`
```

Comment:
Should this change be in this PR? This looks like its for github

Reply:
This change is actually because the `api/alm_settings/list_bindings` API that returns the `GetBinding` object would be incorrect for non-github bindings. For example, the GitLab response does not include the `SummaryCommentEnabled` key in its body and neither would any of the other bindings.